packages/webawesome/docs/.eleventy.js

```diff
@@ -17,6 +17,7 @@ import process from 'process';
 import * as url from 'url';
 import { replaceTextPlugin } from './_plugins/replace-text.js';
 import { searchPlugin } from './_plugins/search.js';
+import { cdnUrl } from './_utils/cdnUrl.js';
```

Comment:
Let's stick with kebab-case files for consistency

```suggestion
import { cdnUrl } from './_utils/cdn-url.js';
```